CHANGES.md

```diff
@@ -1,5 +1,11 @@
 ## [Friend](http://github.com/cemerick/friend) changelog
 
+### `0.3.x`
+
+* HTTP redirect responses now don't enforce absolute URLs - this is a revert of the change made in 0.1.2.
+  See https://github.com/clj-commons/friend/issues/4.
+
```

Comment:
Perhaps make a mention here about the stuff you wrote in the update bit? ie changing the ring middleware?